University-advisor-web/Models/ImageModel.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace University_advisor_web.Models
+{
+    public class ImageModel
+    {
+        public int ImageId { get; set; }
+        public string ImageAddress { get; set; }
+        public string Title { get; set; }
+        public string AuthorCredit { get; set; }
+        public ImageModel(){ }
+        public ImageModel(int iconId, string imageAddress, string title, string authorCredit = "") {
```

Comment:
Just for readability, could you stick to one style of brackets? 
ImageModel()
{

}

Reply:
And one more, is there any reason why you use constructor for setting property values? Properties have public getter and setter, so I guess there is no need to use constructor as in java.

Reply:
Great points, will fix that